src/l10n/ru.yml

```diff
@@ -0,0 +1,61 @@
+airport_charlist: abcdefghijklmnopqrstuvwxyz0123456789 _*%!?#/\\|@
```

Comment:
I have a little suggestion if that's ok. Could you set the Cyrillic alphabet there? It would look super cool during the intro.
These letters are used for the title animation on the home screen

Reply:
Okey, I will do it!

Reply:
Done!